test/test_certdirectory.py

```diff
@@ -261,6 +261,13 @@ def test_list_for_product_match(self):
         res = self.d.list_for_product('123456789')
         self.assertTrue(isinstance(res, list))
 
+    def test_list_for_product_name(self):
+        res = self.d.list_for_product_name('123')
+        self.assertIsInstance(res, list)
+
+    def test_list_for_product_name_case(self):
+        res = self.d.list_for_product_name('123', case_sensitive=False)
+        self.assertIsInstance(res, list)
 
```

Comment:
Please make sure there are 2 empty lines between the last method of the class `EntitlementDirectoryWithCertsTest` and the class `EntitlementDirectoryWithCertsTest`, per Python's PEP 8.

Reply:
Fixed